tests/swe_rise_trans_etc.phpt

```diff
@@ -115,7 +115,7 @@ array(12) {
     ["name"]=>
     string(4) "Moon"
     ["lng"]=>
-    float(221.700155)
+    float(221.700162)
```

Comment:
The changes in this test appear to be minor precision corrections. Let me know if I'm mistaken.